### DISCUSSION TITLE
Release-5.2.0  syntax error in Learning, content,taxonomy,assessment and KP-flinkjob services

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#550] [Bugs]
Hi Team!

Private repo variable( "valid_cloudstorage_base_urls" ) is creating syntax error in the below services.
@santhosh-tg 
 **https://github.com/project-sunbird/sunbird-devops/blob/release-5.2.0/private_repo/ansible/inventory/dev/Core/common.yml#L241**
Learning, 
content,
taxonomy,
assessment and KP-flinkjob services 

Regards,
Gohila

**Comment by santhosh-tg** [comment]
@gohilamariappan  Pls share the logs of the errors. Its most likely that the value for that var is not set in correct format.

**Comment by gohilamariappan** [comment]
Updated value in private repo.


Regards,
Gohila
